website/scripts/build-static-content.js

```diff
@@ -40,6 +41,22 @@ module.exports = {
           } else if (query.resolution === undefined) {
             query.resolution = 'N/A';// « We set this to a string here so that the data type is always string.  We use N/A so folks can see there's no remediation and contribute if desired.
           }
+          if (query.tags){
+            if(!_.isString(query.tags)) {
+              queriesWithProblematicTags.push(query);
+            }
+            // Splitting tags into an array to format them.
+            let tagsToFormat = query.tags.split(',');
+            let formattedTags = [];
+            tagsToFormat.forEach((tag)=>{
```

Comment:
Consider using `for (let tag of tagsToFormat) {}` instead (that's the go-to convention in Sails apps)